packages/client/src/features/tool-feedback/di.config.ts

```diff
@@ -14,48 +14,13 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 import { ContainerModule } from 'inversify';
-import { bindAsService, configureCommand, configureView, LocationPostprocessor, MoveCommand, TYPES } from '~glsp-sprotty';
-import { SResizeHandle } from '../change-bounds/model';
-import { HideChangeBoundsToolResizeFeedbackCommand, ShowChangeBoundsToolResizeFeedbackCommand } from './change-bounds-tool-feedback';
-import { DrawFeedbackEdgeCommand, FeedbackEdgeEnd, RemoveFeedbackEdgeCommand } from './creation-tool-feedback';
-import { ModifyCssFeedbackCommand } from './css-feedback';
-import {
-    DrawFeedbackEdgeSourceCommand,
-    HideEdgeReconnectHandlesFeedbackCommand,
-    ShowEdgeReconnectHandlesFeedbackCommand,
-    SwitchRoutingModeCommand
-} from './edge-edit-tool-feedback';
-import { DrawMarqueeCommand, RemoveMarqueeCommand } from './marquee-tool-feedback';
-import { FeedbackEdgeEndView, SResizeHandleView } from './view';
+import { LocationPostprocessor, MoveCommand, TYPES, bindAsService, configureCommand } from '~glsp-sprotty';
 
 const toolFeedbackModule = new ContainerModule((bind, _unbind, isBound) => {
```

Comment:
Shouldn't t his be part of the `base/feedback`? Since other modules also require the functionality it should not be part of an optional feature module.

Reply:
I moved this around many times so I'm glad to take any good suggestion and that is definitely one! I'll move it.